search-api/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+if [[ $ENVIRONMENT == "PRODUCTION" ]]
+then
+	echo production 
+else
+	pip install ipython
+	pip install -r requirements.txt
```

Comment:
I think requirement installation should be common to both prod and dev, only some requirements will not be installed on prod. So, if that's the case, we can create two requirements.txt files - one for base requirements, and other which are just required for dev env.

Reply:
Yeah will separate out later. And requirements are being installed while building image, installing again so wouldn't have build everytime a requirement changes (only for dev)